src/classes/GetWeather.cls

```diff
@@ -0,0 +1,108 @@
+/**
+ * Created by Mariya on 29.11.2022.
+ */
+
+public with sharing class GetWeather {
```

Comment:
MINOR
Better name would be OpenWeatherDto or OpenWeatherWrapper, or OpenWeatherResponseWrapper as this class serves just as data transfer object.